syncserver/src/web/handlers.rs

```diff
@@ -40,14 +44,35 @@ pub async fn get_collections(
     // The values below, prefixed by `_`, are temporarily and intentionally ignored at present.
     // They will be passed to the Glean logic we will implement to emit metrics.
     // We'd like for the data to be ready and in place to pass to that logic.
-    let _hashed_fxa_uid: String = meta.user_id.hashed_fxa_uid.clone();
-    let _hashed_device_id: String = meta.user_id.hashed_device_id.clone();
+    let hashed_fxa_uid: String = meta.user_id.hashed_fxa_uid.clone();
+    let hashed_device_id: String = meta.user_id.hashed_device_id.clone();
     let user_agent = request
         .headers()
         .get(header::USER_AGENT)
         .and_then(|header| header.to_str().ok())
         .unwrap_or("none");
-    let _device_info = get_device_info(user_agent);
+    let device_info: DeviceInfo = get_device_info(user_agent);
+
+    let logger: GleanEventsLogger = GleanEventsLogger {
+        // app id will be supplied when added to probe-scraper
+        app_id: "syncstorage".to_string(),
+        app_display_version: "1.0.0".to_string(),
+        app_channel: "prod".to_string(),
```

Comment:
We have a `SENTRY_ENVIRONMENT` env var which could be used here but we should probably just add a new `SYNC_ENV` var to the helm chart (like merino has).

Reply:
hardcoding the `app_id` to `syncstorage` wfm, I just mean something else for the `app_channel` of "prod"

Reply:
Got the PR in place to add this, for now we will keep this in place and when the cloudops-infra changes land, we'll modify.